src/utils/request.js

```diff
@@ -51,4 +51,26 @@ const request = extend({
   errorHandler, // extend default error handler with custom actions
 });
 
+request.interceptors.request.use((url, options) => {
+  const token = localStorage.getItem('token');
```

Comment:
As we discussed, this token should probably not be stored in local storage in the browser, just in Javascript memory via a variable.

Reply:
Yes, this won't be stored here. For now I've stored it here for testing purposes. When we are ready, this is gonna be a js variable in memory.